src/graphql/create-user-with-email.ts

```diff
@@ -33,6 +33,10 @@ const createUserWithEmail = {
     if (!args.username) {
       throw `Username can't be an empty string`;
     }
+
+    // TODO: Make a proper fix
+    args.username = args.username.toLowerCase();
+
```

Comment:
```suggestion
```

I'd remove it if the username is not required to be lowercase.

Reply:
we should require usernames to be lower case